content/akeneo-event-platform/akeneo-event-platform.md

```diff
@@ -2,36 +2,11 @@
 
 ## Akeneo Event Platform Overview
 
-The **Akeneo Event Platform (AEP)** seamlessly integrates with Akeneo PIM through a subscription-based event system. External applications can subscribe to specific events triggered within Akeneo PIM, enabling real-time data updates and synchronization.
+The **Akeneo Event Platform (AEP)** seamlessly integrates with Akeneo PIM through a subscription-based event system. External applications can subscribe to specific events triggered within Akeneo PIM, enabling faster data updates and synchronization than continuous polling.
 
 AEP ensures smooth and efficient responses to changes in your PIM, such as product updates, deletions, and other significant events, enhancing your system’s responsiveness and connectivity.
 
-## What AEP Is Not
-
-To set the right expectations, here's what AEP is **not**:
```

Comment:
transformed as FAQ.

Having such "negative" statements in the first page is pretty confusing